### PR TITLE
Testnet node release April 2023

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about changes in the testnet node release for April 2023.

## Changes

Release notes, downloads, previous downloads updated.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
@abizjak Do the configuration files in run-node.rst need to be updated for the new variables?

File: source/mainnet/net/resources/release-notes.rst [hunk 1]
```diff
@@ -11,6 +11,27 @@ Release Notes - Testnet
...
+
+- Improved the node health check, so that if the node is configured with baker credentials, then it is required to be in the baking committee for it to be considered healthy.
+
+- Fixed a bug that could cause the node to hang indefinitely during the out-of-band-catchup when the node is a finalizer.
+
+- Fixed an additional bug in the ``GetAccountInfo`` endpoint in GRPCv2 where the incoming_amounts field of encrypted amounts was not always set correctly.
+
+- The node collector is migrated to a separate package and now uses the V2 GRPC API. For more information, see the Run a node topic that is specific to your node platform: :ref:`Linux<run-a-node>`, :ref:`Ubuntu<run-node-ubuntu-testnet>`, :ref:`Windows<run-node-windows>`, or :ref:`macOS<run-node-macos>`.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:33]: I would more explicitly here that they will need to update their configuration if they have an existing node installed, and then link as you did.

**Comment by abizjak** [comment]
> @abizjak Do the configuration files in run-node.rst need to be updated for the new variables?

Yes, but for the testnet release only the testnet one.

I think the only thing that really needs to change is
```
         - CONCORDIUM_NODE_COLLECTOR_GRPC_HOST=http://testnet-node:10001
```
to
```
         - CONCORDIUM_NODE_COLLECTOR_GRPC_HOST=http://testnet-node:20001
```

**Review by mh-concordium — COMMENTED** [review]
Looks good.

**Pull request merged**